validator/testcases/regex.py

```diff
@@ -1516,6 +1516,36 @@ def tests(self):
             compat_type="error")
 
 
+@register_generator
+class Gecko35RegexTests(CompatRegexTestHelper):
+    """Regex tests for Gecko 35 updates."""
+
+    VERSION = FX35_DEFINITION
+
+    def tests(self):
+        box_object_subject = ("All interfaces derived from nsIBoxObject no "
+                              "longer exist.")
+        yield self.get_test(
+            r"\bnsI[\w\d]*BoxObject\b",
```

Comment:
Probably not necessary. It's what I used to get the list of objects but I don't think anything matches it. I'll remove it.